vdj_ann/src/annotate.rs

```diff
@@ -3018,6 +3018,7 @@ pub struct ContigAnnotation {
     pub invalidated_umis: Option<Vec<String>>, // invalidated UMIs
     pub is_cell: bool,                       // was the barcode declared a cell?
     pub productive: Option<bool>,            // productive?  (null means not full length)
+    pub productive_criteria: Option<ContigStatus>, // New field added in CR 8.1
```

Comment:
Use `///` for documenting fields, so the docs show up in intellisense.  Also, it's best for comments to be "timeless" - prefer describing what something does rather than, for example, when it was added.
```suggestion
    /// <insert description of what this field is>
    pub productive_criteria: Option<ContigStatus>,
```